mne/event.py

```diff
@@ -745,14 +745,20 @@ def make_fixed_length_events(raw, id, start=0, stop=None, duration=1.):
         of the recording.
     duration: float
         The duration to separate events by.
+    first_samp: bool
+        If True (default), times will be adjusted for first_samp. The default
+        behavior is desirable if times are arbitrary, as in resting data. Use 
```

Comment:
I'm thinking of this in the context of resting data, in which there might be triggers to indicate when the participant's eyes were open vs. closed, and create_fixed_length_events would be used to create slices of data from each of those conditions. In that case, setting first_samp=False would also be useful if events will be passed to mne.Epochs, and would be necessary to avoid passing incorrect times to that function (i.e., by adding first_samp again).  How about something like:

> If True (default), times will have raw.first_samp added to them, as in :func:`mne.find_events`. This behavior is not desirable if times were generated by :func:`mne.find_events`.

And then maybe change 
> """Make a set of events separated by a fixed duration 

to

> """Make a set of events separated by a fixed duration. This is useful if events will be passed
to e.g. :class:`mne.Epochs`.

Would that be reasonable?